core/imt/test/testEnableImt.cxx

```diff
@@ -0,0 +1,26 @@
+#include "TROOT.h"
+#include "ROOT/RTaskArena.hxx"
+
+#include "tbb/task_arena.h"
+
+#include "ROOT/TestSupport.hxx"
+#include "gtest/gtest.h"
+
+#ifdef R__USE_IMT
+
+static const unsigned gMaxConcurrency = ROOT::Internal::LogicalCPUBandwidthControl();
+
+TEST(EnableImt, TBBAttach)
+{
+   tbb::task_arena main_arena;
+   main_arena.initialize(2);
+
+   ROOT::EnableImplicitMT(ROOT::EIMTConfig::kExistingTBBArena);
+
+   auto psize = ROOT::GetThreadPoolSize();
+
+   EXPECT_TRUE( psize > 1);
+   EXPECT_EQ( psize, 2); // gMaxConcurrency);
```

Comment:
@Dr15Jones This test fails.  So I doing something wrong either here (line 15/16) or in RTaskArena.cxx line 118/125

Reply:
I suggest also doing `EXPECT_TRUE( main_arena.max_concurrency() == 2);`